src/main/java/com/savvato/tribeapp/services/UserServiceImpl.java

```diff
@@ -162,10 +163,26 @@ private UserDTO getUserDTO(User user) {
 				.enabled(user.getEnabled())
 				.created(user.getCreated().toString())
 				.lastUpdated(user.getLastUpdated().toString())
-				.roles(user.getRoles())
+				.roles(getUserRoleDTO(user))
 				.build();
-
 		return userDTO;
 	}
+	private List <UserRoleDTO> getUserRoleDTO(User user){
+		Set<UserRole> userRole = user.getRoles();
+		List<UserRoleDTO> rtn = new ArrayList<>();
+		Iterator<UserRole> iterator  = userRole.iterator();
+        while (iterator.hasNext()){
+            UserRole userRoles = iterator.next();
```

Comment:
just for clarity, you should name this 'ur' or 'userRole' instead of the plural form